PT.md

```diff
@@ -36,4 +36,10 @@ Uma lista de podcasts que podem ser úteis para programadores e engenheiros de s
 * [ZOFE](https://zofe.com.br/)
   * **Descrição**: Tópicos gerais sobre desenvolvimento de software, frontend, javascript
   * **Frequência**: Normalmente 2 vezes por mês
-  * **Duração**: 30 - 45 min
\ No newline at end of file
+  * **Duração**: 30 - 45 min
+  
+  
+* [Hipsters Ponto Tech](https://hipsters.tech/)
```

Comment:
Not in sorted order

Reply:
Sorry, but I don't understand what you want to mean.